AWS/DirectoryInsights/get-jcdirectoryinsights.py

```diff
@@ -0,0 +1,76 @@
+import requests, datetime, json, boto3, os, gzip
```

Comment:
consider adding a requirements.txt. you can install deps from there. pip install -r requirements.txt

Reply:
Would this comment be more for the README if they wanted to build and deploy this manually? It is my understanding that with a Lambda function you have to package all this stuff up beforehand when you publish it. Definitely a good note though I should make sure that I freeze the versions that I'm currently using and get that added.

Reply:
Yes, its more for the readme, just to make it simple. We a have special zip and deploy process because we have come across errors building locally (macos) and uploading into the lambda env. Building in an amazonlinux container solved those issues. https://github.com/TheJumpCloud/data-engineering-lambdas#deployment

Reply:
Oooo. That deploy script sounds interesting. It looks like I don't have access to it though.

Since you mentioned the requirements.txt, I also got to thinking about potentially using a setup.py as well. I am definitely going to add the requirements.txt in but might explore a setup script as a fast follow. Will resolve this convo when I get that added in.

Reply:
requirements.txt is now included.